modules/s3-avrae/main.tf

```
@@ -41,6 +41,17 @@ POLICY
   )
 }
 
+# lambda pre-deploy bucket
+resource "aws_s3_bucket" "lambda_archives" {
+  bucket = "${var.s3_prefix}-${var.region}-${var.service}-${var.env}-lambda-archives"
+  acl = "private"
+  tags = merge(
+  local.common_tags,
+  {
+    "Name" = "${var.service}-lambda-archives"
+  })
+}
+
```

```suggestion

resource "aws_s3_bucket_public_access_block" "lambda_archives" {
  bucket                  = aws_s3_bucket.lambda_archives.id
  block_public_acls       = true
  block_public_policy     = true
  ignore_public_acls      = true
  restrict_public_buckets = true
}
```
Insurance policy against accidental public items.